configs/rtmdet/rotated/rtmdet-r_l_syncbn_fast_1xb8_36e_dota.py

```diff
@@ -0,0 +1,286 @@
+_base_ = '../../_base_/default_runtime.py'
+
+checkpoint = 'https://download.openmmlab.com/mmdetection/v3.0/rtmdet/cspnext_rsb_pretrain/cspnext-l_8xb256-rsb-a1-600e_in1k-6a760974.pth'  # noqa
+
+data_root = '/datasets/dota_mmrotate_ss/'
```

Comment:
Relevant data processing script training steps should also be briefly explained in the readme